website/docs/r/token_auth_backend_role.html.md

```diff
@@ -35,8 +35,12 @@ The following arguments are supported:
 
 * `allowed_policies` (Optional) List of allowed policies for given role.
 
+* `allowed_policies_glob` (Optional) List of allowed policies with glob match for given role.
```

Comment:
nit: should probably be documented as a `set` rather than `list`
```suggestion
* `allowed_policies_glob` (Optional) Set of allowed policies with glob match for given role.
```